### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by salleq**
Added example for using docker-compose

### CODE REVIEW

**Comment by salleq** [comment]
Hope I did it right!

**Comment by salleq** [comment]
I added docker-compose for deepstack itself, too.

"Localstorage" should be declared in docker-compose to get it working so I entered just some directory instead.

File: README.md [hunk 1]
```diff
@@ -43,9 +43,35 @@ robmarkcole/deepstack-ui:latest`
...
+      - VISION-DETECTION=True
+      - API-KEY="my_key"
+    volumes:
+      - /srv/docker/deepstack:/datastore
+  deepstack_ui:
+    container_name: deepstack_ui
+    restart: unless-stopped
+    image: robmarkcole/deepstack-ui:latest
+    environment: 
+      - DEEPSTACK_IP=192.168.1.133
```
Comment (robmarkcole) [inline README.md:64]: There should be a way to pass this in without hard coding?

Comment (salleq) [reply README.md:64]: Can you clarify a bit - which part ? 
Environment variables or ... ?

Comment (robmarkcole) [reply README.md:64]: The IP address https://docs.docker.com/compose/networking/

Comment (salleq) [reply README.md:64]: Ah okay so you mean that you should be able to set a variable in the "deepstack ui" -part of the docker-compose to point at the IP of deepstack without hard coding the IP. 

Right, that can be done by just pointing to the deepstack container, I just tested it works and I'll do the change in the readme.

Comment (robmarkcole) [reply README.md]: great!

**Pull request merged**